BaseFilter/sections/antiadblock.txt

```diff
@@ -7590,8 +7590,7 @@ thoptv.info#%#AG_onLoad(function() { var el=document.querySelector('ins.adsbygoo
 !+ PLATFORM(ios, ext_android_cb)
 @@||freevpn.us^$generichide
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26212
-uploadbox.io$$script[tag-content="replaceContentWithAdBlockerContent"][max-length="1500"]
```

Comment:
Looks like you can keep this rule - it still works fine.

Reply:
Please just a sec.